docs/revsets.md

```diff
@@ -78,96 +78,96 @@ only symbols.
 You can use parentheses to control evaluation order, such as `(x & y) | z` or
 `x & (y | z)`.
 
-??? examples
+### Examples
```

Comment:
I think the current form is a compromise to make it render well on the web page (using MkDocs, iirc). I think I may have sent a similar PR once. Maybe we need to add a comment here so people don't waste time trying to "fix" it.

Reply:
Good idea. I'll propose a comment instead.

Reply:
Done.